onnx/checker.h

```diff
@@ -27,12 +28,27 @@ class ValidationError : public std::runtime_error {
 #define fail_check(...) \
   throw onnx::checker::ValidationError(onnx::MakeString(__VA_ARGS__));
 
+struct CheckerContext {
+  int ir_version;
```

Comment:
Given we have getter/setter for these fields, shall we use "class" and keep them as private please?

Reply:
OK